src/jsifier.js

```diff
@@ -480,13 +480,8 @@ function ${name}(${args}) {
     const shellFile = MINIMAL_RUNTIME ? 'shell_minimal.js' : 'shell.js';
     print(processMacros(preprocess(read(shellFile), shellFile)));
 
-    let pre;
-    if (MINIMAL_RUNTIME) {
-      pre = processMacros(preprocess(read('preamble_minimal.js'), 'preamble_minimal.js'));
-    } else {
-      pre = processMacros(preprocess(read('support.js'), 'support.js')) +
-            processMacros(preprocess(read('preamble.js'), 'preamble.js'));
-    }
+    const preFile = MINIMAL_RUNTIME ? 'preamble_minimal.js' : 'preamble.js';
+    const pre = processMacros(preprocess(read(preFile), preFile));
```

Comment:
Can you just do `print(processMacros..` and avoid the `pre` variable completely?

Reply:
Ah, that is indeed possible/neater. Should we do the same for `postamble{,_minimal}.js` too?:
https://github.com/emscripten-core/emscripten/blob/056ee9495cf2bee5a0d6f69536ff09b6d6e05283/src/jsifier.js#L527-L529

Reply:
(I could also unify this as a follow-up, if you want)

Reply:
followup sounds good.

Reply:
#18422 :)